### DISCUSSION TITLE
RattlerSPORE Discussions

### DISCUSSION THREAD

**Discussion by NeDether** [NeDether/Rattler-Spore#1] [Announcements]
## 🔧😁    Howdy!
You can use this discussion tab to:
  * Suggest ideas for the mod.
  * Ask for help or support with the mod.
  * Engage with other community members.

**Comment by LesterDesicle** [comment]
Will be glad having guide how Ware crafted items works in the mod.

**Comment by starturtle27** [comment]
Some ideas I had for the mod

1.) Have the changes to the planet's terrain save, instead of just reverting back (don't really know if this is possible)

2.) If you can do the above, then a beam that adds terrain would be cool for more precise terraforming

I hope you consider adding my ideas to your mod.

**Comment by Higor-Ssjzus** [comment]
How does Mineral Scanner Works?

**Reply by NeDether** [reply]
The mineral scanner's UI does not show up on lower screen resolutions. The screen size must scale up to at least around 1980x1200, this is a bug that I will patch within either another miniature update or the update that implements factories.
